rest/src/main/groovy/whelk/rest/api/Crud.groovy

```diff
@@ -92,6 +96,7 @@ class Crud extends HttpServlet {
         vocabData = whelk.vocabData
         jsonld = whelk.jsonld
         search = new SearchUtils(whelk)
+        validator = JsonLdValidator.from(jsonld)
```

Comment:
Need to stop processing if validation fails, i.e. return.
I think it's cleaner to throw and write error response at top level.

Reply:
Oops!